src/app/shared/model/distribution.model.spec.ts

```diff
@@ -8,37 +8,46 @@ describe('DistributionModel', () => {
   let series = cms.mock('prx:series', {id: 'series1'});
   let fooDist = series.mock('prx:distributions', {id: 'dist1', kind: 'foo'});
   let podDist = series.mock('prx:distributions', {id: 'dist2', kind: 'podcast', url: podcastUrl});
+  let seriesVersionTemplates = series.mockItems('prx:audio-version-templates', []);
+  let seriesVersionFileTemplates = series.mockItems('prx:audio-file-templates', [{}]);
+  let seriesVersionFileTemplateItems = seriesVersionFileTemplates[0].mockItems('prx:items', [{}]);
+  let fooDistTemplate = fooDist.mock('prx:audio-version-template', {id: 'templateFoo'});
+  let fooDistFileTemplate = fooDistTemplate.mock('prx:audio-file-templates', {id: 'fileFoo'});
+  let fooDistFileTemplateItems = fooDistFileTemplate.mockItems('prx:items', [{}]);
```

Comment:
Something about this is mocked incorrectly, throwing error `Expected mocked array at prx:items - got object`